eclipse-scout-core/src/carousel/Carousel.ts

```diff
@@ -58,7 +58,9 @@ export class Carousel extends Widget implements CarouselModel {
 
   protected override _render() {
     // add container
-    this.$container = this.$parent.appendDiv('carousel');
+    this.$container = this.$parent.appendDiv('carousel')
+      // Tooltips on elements in the filmstrip will check all scroll parents (i.e. elements marked with 'scrollable') if the anchor is visible.
+      .data('scrollable', true);
```

Comment:
Unfortunately, that has an unexpected side effect. When opening the Carousel page in the JsWidgets app, and then selecting a different page, an error occurs:

> [ERROR] scrollable could not be removed. Potential memory leak. carousel field alternative
>     at Object.removeScrollable
>     at removeScrollable
>     at uninstall
>     at _uninstallScrollbars
>     at _cleanup
>     at _removeInternal
>     ...
>